### PR TITLE
Wrote The Talon Concepts -> Settings Section

### PR DESCRIPTION
**Pull request by AndrewDant**

### CODE REVIEW

File: unofficial_talon_docs.md [hunk 1]
```diff
@@ -727,7 +727,44 @@ You may have noticed that scopes can emulate the behaviour of [tags](#tags), exc
...
+from talon import Module
+
+mod = Module()
+
+horizontal_position = mod.setting(
+    "my_package_horizontal_position",
+    type=int,
+    default=0
+    desc="Set the horizontal display position of some UI element",
+)
```
Comment (splondike) [inline unofficial_talon_docs.md:745]: Should also include how to read the setting out (`horizontal_position.get()`).

**Review by splondike — COMMENTED** [review]
Looks good, thanks! Just that one thing to add in I think.

I also noticed the use of 'talon user file set' ;).

**Review by tararoys — CHANGES_REQUESTED** [review]
Line 743 is missing a comma at the end of `default=0`

Apart from that, a useful overview!  

It *might* be worth mentioning that setting ocks are fragile compared to the rest of Talon's syntax.

In Talon files, an error anywhere in the settings block will disable every setting that has been set in that settings block.  (I tested this by introducing errors into the settings block of my dense mouse grid and a syntax setting change anywhere in the block made it revert back to my defaults.)

**Comment by splondike** [comment]
That fragility may be a feature. I imagine you could want the whole group of settings to apply atomically, if it were partially applied you might get unexpected results.

Happy with either having it in or not as per Andrew's preference.

**Comment by AndrewDant** [comment]
I've implemented the changes suggested by [tararoys](https://github.com/tararoys)

**Comment by splondike** [comment]
Thanks!

**Pull request merged**